src/main/java/com/stosik/parking/reservation/domain/ReservationConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package com.stosik.parking.reservation.domain;
+
+import com.stosik.parking.reservation.domain.evaluator.PriceCalculator;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+class ReservationConfiguration
+{
+    @Bean
+    ReservationFacade reservationFacade(ReservationRepository reservationRepository, ParkingStore parkingStore, Meter parkingMeter)
+    {
+        PriceCalculator priceCalculator = new PriceCalculator();
+        ReservationDtoCreator reservationDtoCreator = new ReservationDtoCreator();
```

Comment:
dlaczego niektóre rzeczy są beanami (ReservationRepository, ParkingStore, Meter) a inne nie są (PriceCalculator, RevervationDtoCreator)?

Reply:
Glownie zalozenie jakie przyswiecalo mi od samego poczatku to jedynie zaleznosci ktore beda mialy bezposrednio doczynienie z I/O lub je imitowaly rejestrowac jako beany, pozostale zaleznosci tworzyc normalnie, ew. gdyby w przyszlosci pojawil sie kolejny modul to rowniez jego fasade wstrzykiwac jako zaleznosc (port & adapters)